zvariant/src/dict.rs

```diff
@@ -1,3 +1,6 @@
+use std::collections::btree_map::IntoIter as BTreeMapIntoIter;
+use std::collections::btree_map::Iter as BTreeMapIter;
+use std::collections::btree_map::IterMut as BTreeMapIterMut;
```

Comment:
* This will fail the fmt job because of `imports_granularity = "Crate"` in our `.rustfmt.toml`. Please run `cargo +nightly fmt` and `cargo clippy` (apart from `cargo test`) before pushing your changes to avoid ping-pongs with the CI. :)

* I don't even see them being used in this file. Did you check if this builds (w/o warnings)? Perhaps I'm being blind? :thinking:

Reply:
Updated according to format and clippy.